draft-specifications/CapTP Specification.md

```diff
@@ -127,42 +127,43 @@ convention is followed for the CapTP bootstrap object and for promises.
+they become settled by either being fulfilled with a value or broken with an
+error.
 
-Promises are often created by sending an `op:deliver` message, where they
-represent the eventual value of the response. They can be chained together in
-what is called [Promise Pipelining](#promise-pipelining), whereby messages are
-sent to the promise which should be delivered to its resolution value if it is
-fulfilled with a single (as opposed to breaking with an error, or fulfilled with
-multiple values).
+Promises are often created for representing the eventual response to an
+[`op:deliver`](#opdeliver) message. They can be chained together in what is
+called [Promise Pipelining](#promise-pipelining), whereby messages are sent
+directly to a promise as a proxy for its eventual fulfillment value.
 
-## [Promise and Resolver Objects](#promise-objects)
+<a id="promise-objects"></a>
+## [Promise and Resolver Objects](#promise-and-resolver-objects)
 
 Promises work like regular objects in CapTP. Promises come as a pair:
 
-- The promise object itself which represents a value.
-- The resolver object which is used to provide the promise with its resolved
-  value, or break it in the case of an error.
-
- The promise object may eventually resolve to either a concrete value, object
- reference, another promise (in the case of promise pipelining), or may break.
- When a promise breaks its resolved with an error, breakages can be caused by
- either explicit instruction, by implicit error propagation, or network
- partition.
-
-Promises can be listened to with the [`op:listen`](#oplisten) operation, or
-messages can be sent to them as if it were the resolved object. The messages
-will be relayed to the eventual object if it is `fulfill`ed to one. If the
-promise instead `break`s and thus has no resolved object, messages cannot be
-delivered and promises created during the sending of those messages should also
-break.
+- The promise object itself which represents an eventual value (usually sent in
+  responses, and also with promise pipelining as the target of almost any
+  message).
+- The resolver object which is notified to provide the promise with its
+  fulfillment value or break it in the case of an error (usually sent in
+  requests, notably including [`op:deliver`](#opdeliver) and
+  [`op:listen`](#oplisten)).
+
+A promise object can remain pending indefinitely, but usually either fulfills
+to a [Value][Model-Passable] (which in the case of promise pipelining can
+itself be a promise) or breaks with an error. Breakages can be caused by either
+explicit instruction, implicit error propagation, or network partition.
+
+Promises can be listened to with the [`op:listen`](#oplisten) operation, and
+messages can be sent to them as a proxy for their eventual fulfillment value.
+Upon fulfillment, such messages are relayed to that value. But if the promise
+breaks or fulfills to a non-[Reference][Model-Reference] value, the messages
```

Comment:
The only fulfillment value that handles deliveries is currently Target. Promise isn’t a fulfillment value. We’re on the fence about whether Error is a Reference, capdata, or hybrid. Sturdy Reference will fall under Reference and we have not reached a consensus on delivery semantics for them, and there are multiple good options.
```suggestion
breaks or fulfills to a non-[Target][Model-Target] value, the messages
```